modules/rostests/apitests/imm32/JapanImeConvTest.h

```diff
@@ -0,0 +1,259 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for Japanese IME conversion
+ * COPYRIGHT:   Copyright 2022 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include <ntstatus.h>
+#define WIN32_NO_STATUS
+#include <windows.h>
+#include <windowsx.h>
+#include <imm.h>
+//#include <ddk/immdev.h>
+#include <wine/test.h>
+//#include <pseh/pseh2.h>
+#include <stdio.h>
```

Comment:
Move before ntstatus and other inclusions.

Reply:
Deleted.